sdk/appcenter-storage/src/main/java/com/microsoft/appcenter/storage/LocalDocumentStorage.java

```diff
@@ -111,10 +122,11 @@
 
     private LocalDocumentStorage(DatabaseManager databaseManager) {
         mDatabaseManager = databaseManager;
+        createUserTable();
```

Comment:
user_null_documents is created on my test device when signed out, accountId must be tested before creating

Reply:
I got confused by when `onCreated` gets called (which on DB creation, not every time the object gets initialized), which is what forced me to not add the existence check in the first place. Now added `IF NOT EXISTS` which would work for both when the DB gets created and when a new table is created within an existing DB.

Reply:
That's not the issue here. We are not signed in and we create a table for `null`.